packages/react/src/components/select/Select.module.scss

```diff
@@ -252,23 +255,15 @@
 
         &.disabledOption.disabledOption {
           cursor: not-allowed;
-
-          &:hover {
-            background-color: transparent;
-          }
         }
       }
 
       .singleSelectListItem {
-        --menu-item-background-hover: var(--color-bus-light);
-        --menu-item-background-selected: var(--color-bus);
-        --menu-item-background-selected-hover: var(--color-bus);
-        --menu-item-color-selected: var(--color-white);
-        --menu-item-color-selected-hover: var(--color-white);
-
-        &.disabledOption.disabledOption {
-          color: var(--color-black-40);
-        }
+        --computed-menu-item-background-hover: var(--menu-item-background-hover, var(--color-bus-light));
```

Comment:
hmm... should all these `*-background-*` be `*-background-color-*`? and if there's more of these in which it only affects a -color?

Reply:
The naming is copied from the old Select, so haven't really thought about it. The only thing that can change in the background is the color.

Reply:
So then I'd prefer it to be *-color* :D everywhere

Reply:
Changed!